static/index.html

```diff
@@ -240,7 +245,10 @@ <h4>Print Progress</h4>
                                                     <div class="col-6">
                                                         <div class="input-group">
                                                             <div class="input-group-prepend">
-                                                                <div class="input-group-text" id="timeElapsed">
+                                                                <div
+                                                                    class="input-group-text print-info-label"
```

Comment:
We're adding this specific class to a number of elements here, and they all seem to be selectable by a rule such as `input-group-prepend input-group-text` or similar.

Do we really need a new css class here?

Reply:
> We're adding this specific class to a number of elements here, and they all seem to be selectable by a rule such as `input-group-prepend input-group-text` or similar.
> 
> Do we really need a new css class here?

I did this so that we only effected these fields and not any other field also using the `input-group-text` class (which is used on the Upload form for the config file.